Description: Removes u'' and u"" from upstream source
 Upstream is using some unicode strings which are incompatible with python 3.2,
 this patch removes them.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2014-02-06

--- python-pbr-0.6.orig/pbr/packaging.py
+++ python-pbr-0.6/pbr/packaging.py
@@ -270,7 +270,7 @@ def write_git_changelog(git_dir=None, de
             first_line = True
             with io.open(new_changelog, "w",
                          encoding="utf-8") as changelog_file:
-                changelog_file.write(u"CHANGES\n=======\n\n")
+                changelog_file.write("CHANGES\n=======\n\n")
                 for line in changelog.split('\n'):
                     line_parts = line.split()
                     if len(line_parts) < 2:
@@ -291,9 +291,9 @@ def write_git_changelog(git_dir=None, de
 
                         underline = len(tag) * '-'
                         if not first_line:
-                            changelog_file.write(u'\n')
+                            changelog_file.write('\n')
                         changelog_file.write(
-                            (u"%(tag)s\n%(underline)s\n\n" %
+                            ("%(tag)s\n%(underline)s\n\n" %
                              dict(tag=tag,
                                   underline=underline)))
 
@@ -301,7 +301,7 @@ def write_git_changelog(git_dir=None, de
                         if msg.endswith("."):
                             msg = msg[:-1]
                         changelog_file.write(
-                            (u"* %(msg)s\n" % dict(msg=msg)))
+                            ("* %(msg)s\n" % dict(msg=msg)))
                     first_line = False
 
 
--- python-pbr-0.6.orig/pbr/tests/test_setup.py
+++ python-pbr-0.6/pbr/tests/test_setup.py
@@ -145,10 +145,10 @@ class GitLogsTest(base.BaseTestCase):
             self.assertNotIn('Merge "', changelog_contents)
 
     def test_generate_authors(self):
-        author_old = u"Foo Foo <email@foo.com>"
-        author_new = u"Bar Bar <email@bar.com>"
-        co_author = u"Foo Bar <foo@bar.com>"
-        co_author_by = u"Co-authored-by: " + co_author
+        author_old = "Foo Foo <email@foo.com>"
+        author_new = "Bar Bar <email@bar.com>"
+        co_author = "Foo Bar <foo@bar.com>"
+        co_author_by = "Co-authored-by: " + co_author
 
         git_log_cmd = (
             "git --git-dir=%s log --use-mailmap --format=%%aN <%%aE>"
